modules/installation-requirements-user-infra.adoc

```diff
@@ -80,6 +102,65 @@ Each cluster machine must meet the following minimum requirements:
 
 |===
 
+ifeval::["{context}" == "installing-ibm-z"]
```

Comment:
I would split this section into a separate module, add a few more context statements about z in general, and put it higher in the assembly.